packages/roosterjs-editor-plugins/test/TableResize/tableResizeTest.ts

```diff
@@ -0,0 +1,135 @@
+        const editorDiv = editor.getDocument().getElementById(TEST_ID);
+
+        let mouseEvent = new MouseEvent('mousemove', {
+            clientX: mouseStart.x,
+            clientY: mouseStart.y,
+        });
+
+        editorDiv.dispatchEvent(mouseEvent);
+
+        const body = editor.getDocument().body;
+
+        expect(body.innerHTML.includes(ADD_BUTTON)).toBe(true);
+
+        mouseEvent = new MouseEvent('mousemove', {
+            clientX: mouseEnd.x,
+            clientY: mouseEnd.y,
+        });
+        editorDiv.dispatchEvent(mouseEvent);
+        expect(body.innerHTML.includes(ADD_BUTTON)).toBe(expectedResult);
+    }
+
+    it('removes the vertical inserter when moving the cursor out of the offset zone', () => {
+        const rect = initialize();
+        runTest({ x: rect.right, y: rect.top }, { x: rect.right / 2, y: rect.bottom }, false);
+    });
+
+    it('keeps the vertical inserter when moving the cursor inside the safe zone', () => {
+        const rect = initialize();
+        runTest(
+            { x: rect.right, y: rect.top },
+            { x: rect.right - insideTheOffset, y: rect.top },
+            true
+        );
+    });
+
+    it('removes the horizontal inserter when moving the cursor out of the offset zone', () => {
+        const rect = initialize();
+        runTest(
+            { x: rect.left, y: rect.bottom },
+            { x: (rect.right - rect.left) / 2, y: (rect.bottom - rect.top) / 2 },
+            false
+        );
+    });
+
+    it('keeps the horizontal inserter when moving the cursor inside the safe zone', () => {
+        const rect = initialize();
+        runTest(
+            { x: rect.left, y: rect.bottom },
+            { x: rect.left, y: rect.bottom - insideTheOffset },
+            true
+        );
+    });
+
+    it('removes the vertical inserter when moving the cursor out of the offset zone with culture language RTL', () => {
+        const rect = initialize(true);
+        runTest(
+            { x: rect.left, y: rect.top },
+            { x: (rect.right - rect.left) / 2, y: rect.bottom },
+            false
+        );
+    });
+
+    it('keeps the vertical inserter when moving the cursor inside the safe zone with culture language RTL', () => {
+        const rect = initialize(true);
+        runTest(
+            { x: rect.left, y: rect.top },
+            { x: rect.left, y: rect.top + insideTheOffset },
+            true
+        );
+    });
+
+    it('removes the horizontal inserter when moving the cursor out of the offset zone with culture language RTL', () => {
+        const rect = initialize(true);
+        runTest(
+            { x: rect.right, y: rect.bottom },
+            { x: (rect.right - rect.left) / 2, y: (rect.bottom - rect.top) / 2 },
+            false
+        );
+    });
+
+    it('keeps the horitzontal inserter when moving the cursor inside the safe zone with culture language RTL', () => {
+        const rect = initialize(true);
+        runTest(
```

Comment:
Since we don't show horizontal inserter for the very first line, rect.top will fail this test, using rect.bottom should fix it. 
 ```
runTest(
            { x: rect.right, y: rect.bottom },
            { x: rect.right + insideTheOffset, y: rect.bottom },
            true
        );
```

Reply:
Good catch!, I was using the wrong coordinates here. I will fix this using the right+bottom corner to test the horizontal inserter in RTL mode.